mx.truffleruby/mx_truffleruby.py

```diff
@@ -118,17 +118,25 @@ def ruby_testdownstream_hello(args):
     mx.run(['bin/truffleruby', '-e', 'puts "Hello Ruby!"'])
 
 def ruby_testdownstream_aot(args):
-    if len(args) != 1:
-        mx.abort("Missing path to AOT image: mx ruby_testdownstream_aot <aot_bin>")
+    if len(args) > 3:
+        mx.abort("Incorrect argument count: mx ruby_testdownstream_aot <aot_bin> [<format>] [<build_type>]")
```

Comment:
There shouldn't be any difference in the build type, but in practice we're having problems with the AOT debug builds and I don't want to restrict the set of tests we run for production builds as a result. I went this way because if I pass the arguments straight through to control the set of tests to run, we're back to the losing control over which tests we run.

I think allowing people running downstream tests to use a less verbose format than the 'spec' we normally use has value.